varnish-operator/values.yaml

```diff
@@ -1,8 +1,12 @@
 # Number of pods running the operator
 replicas: 1
 container:
-  # path to image + tag
-  image: ibmcom/varnish-operator:0.34.0
+  # path to image + tag, for reverse compatibility
+  # image: ibmcom/varnish-operator
+  registry: ibmcom
+  repository: varnish-operator
+  tag: 0.34.0
+  digest: ""
```

Comment:
```suggestion
  # Only specify the digest if you want to use it instead of the tag to specify what
  # version of the image to pull.
  digest: ""
```